website/project/model.py

```diff
@@ -3782,6 +3782,30 @@ def to_json(self):
             "text": self.text
         }
 
+
+class DraftRegistrationLog(StoredObject):
+    """ Simple log to show status changes for DraftRegistrations
+
+    field - _id - primary key
+    field - date - date of the action took place
+    field - action - simple action to track what happened
+    """
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+    date = fields.DateTimeField(default=datetime.datetime.utcnow, index=True)
+    action = fields.StringField(index=True)
+
+    @classmethod
+    def add_log(cls, action):
+        log = cls(
+            action=action,
+        )
+        log.save()
+        return log
+
+    def __repr__(self):
+        return 'DraftRegistrationLog: {}'.format(self._id)
```

Comment:
This is ok, but it should be consistent with other `__repr__` methods. It would also be helpful to include the date and action.

It should look something like

```
<DraftRegistrationLog(date=datetime.datetime(...), action='foo') with _id 'abc12'>
```